resources/event-bus/charts/event-publish-service/templates/migration-jobs.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "event-publish-service.fullname" . }}-hooks
+  annotations:
+    helm.sh/hook: pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "event-publish-service.fullname" . }}-hooks
+  annotations:
+    helm.sh/hook: pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kyma-admin
+subjects:
+- kind: ServiceAccount
+  name: {{ template "event-publish-service.fullname" . }}-hooks
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "event-publish-service.fullname" . }}-pre-upgrade
+  labels:
+{{ include "event-publish-service.labels.standard" . | indent 4 }}
+  annotations:
+    helm.sh/hook: pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: 'false'
+      labels:
+{{ include "event-publish-service.labels.standard" . | indent 8 }}
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ template "event-publish-service.fullname" . }}-hooks
+      containers:
+      - name: pre-upgrade
+        image: docker.io/istio/kubectl:1.2.3
```

Comment:
is there a reason we use this image instead of eu.gcr.io/kyma-project/test-infra/alpine-kubectl ?

Reply:
Good question, different components seem to be using different images quite inconsistently.

Reply:
Shot in the dark: @Demonsthere do the know what the best practice is?